src/zero.ts

```diff
@@ -0,0 +1,56 @@
+import { Response } from "./types";
+import { APIError, APIResultError } from "./errors";
+
+declare const fetch: any; // tslint:disable-line no-any
+
+/**
+ * Zero is a  client to a Dgraph Zero instance.
+ * This is a seperate class because it may be removed in the future.
+ */
+export class DgraphZero {
+    private readonly addr: string;
+
+    /**
+     * Creates a new Client for interacting with Dgraph Zero.
+     *
+     */
+    constructor(addr?: string) {
+        if (addr === undefined) {
+            this.addr = "http://localhost:6080"; // tslint:disable-line no-http-string
```

Comment:
one day we should start using tslint:disable-next-line so the code can be read without horizontal scroll, but probably not today.